client/game-page/src/main/kotlin/com/bytelegend/client/app/ui/CoinCountWidget.kt

```diff
@@ -40,6 +41,14 @@ class CoinCountWidget : AbstractIncrementAnimatableWidget<CoinCountWidgetProps,
             +game.heroPlayer.coin.toString()
         }
         renderIcon()
+        attrs.onClickFunction = {
+            game.modalController.show {
+                attrs.className = "history-modal"
```

Comment:
We don't need this right now. If we have to, it would be better to put into `HistoricalModal` to avoid duplicate code in 3 places. Now let's just remove this class.